server/e2e/main_test.go

```diff
@@ -0,0 +1,66 @@
+//go:build e2e
+// +build e2e
+
+package main
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/mattermost/mattermost-plugin-msteams-sync/server/msteams"
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/pelletier/go-toml/v2"
+	"github.com/stretchr/testify/require"
+)
+
+type TestConfig struct {
+	MSTeams struct {
+		TenantID               string `toml:"tenant_id"`
+		ClientID               string `toml:"client_id"`
+		ConnectedChannelTeamID string `toml:"connected_channel_team_id"`
+		ConnectedChannelID     string `toml:"connected_channel_id"`
+		ChatID                 string `toml:"chat_id"`
+	} `toml:"msteams"`
+	Mattermost struct {
+		URL                string `toml:"url"`
+		UserUsername       string `toml:"user_username"`
+		UserPassword       string `toml:"user_password"`
+		AdminUsername      string `toml:"admin_username"`
+		AdminPassword      string `toml:"admin_password"`
+		ConnectedChannelID string `toml:"connected_channel_id"`
+		DmID               string `toml:"dm_id"`
+	} `toml:"mattermost"`
+}
+
+var msClient msteams.Client
+var mmClient *model.Client4
+var mmClientAdmin *model.Client4
+var testCfg *TestConfig
+
+func setup(t *testing.T) {
+	if testCfg == nil {
+		data, err := os.ReadFile("testconfig.toml")
```

Comment:
Nit:  would be nice if this error lead people to a small README that explained how to set this up.